src/ralph/accounts/admin.py

```diff
@@ -91,6 +92,21 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class StaffStatusFilter(BooleanFilter):
```

Comment:
It's :ok: for me.
But in some filters we have greater logic.
Maybe generator classes for filters? RalphFilter(name, params, type) class return (SuperuserFilter(BooleanFilter))

@ar4s, @mkurek, @xliiv What do you think?